src/main/java/uk/gov/ons/ctp/response/sample/domain/model/CollectionExerciseJob.java

```diff
@@ -56,4 +56,7 @@
 
   @Column(name = "samplesummaryid")
   private UUID sampleSummaryId;
+
+  @Column(name = "jobcomplete")
+  private boolean jobComplete;
```

Comment:
Does the CollectionExerciseJobCreationRequestDTO (and other CollectionExerciseJob DTOs) need changing?

Reply:
No... the flag is for internal use within the service. Good spot though